modules/twinkleblock.js

```diff
@@ -0,0 +1,879 @@
+//<nowiki>
+
+
+(function($){
+
+var api = new mw.Api(), result, relevantUser = mw.config.get('wgRelevantUserName'), relevantUserName, formCleared = false;
```

Comment:
I don't think any of these are required, or at least shouldn't be. e.g. no need to cache relevantUser like this.